test/services/location_service_test.dart

```diff
@@ -0,0 +1,93 @@
+
+import 'package:concordia_navigation/models/user_location.dart';
+import 'package:concordia_navigation/services/location_service.dart';
+import 'package:flutter/services.dart' show MethodCall, MethodChannel;
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:location/location.dart';
+import 'package:mockito/mockito.dart';
+
+class MockLocation extends Mock implements Location{}
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+  group('LocationService', () {
+    
+    Location location;
```

Comment:
Next time, try using `git mv location_search_test.dart location_service_test.dart`. This way we can easily see if something was moved rather than deleted and recreated but with a different name

Reply:
did not know that
thanks